front/src/usermanagement/UserManagementView.js

```diff
@@ -481,15 +575,16 @@ class UserManagementView extends Component {
 
   update() {
     const tempRows = [];
-    this.state.userList.forEach((user) => {
-      if (this.state.username !== user.name) {
-        const row = this.createData(user.name,
-          user.role,
-          this.returnPassButton(user.id, manage, fin),
-          this.returnRemoveButton(user.id, manage, fin));
+    for (const i in this.state.userList) {
+      if (localStorage.taseraUserName !== this.state.userList[i].name) {
```

Comment:
This will not work with cookies, check compatibility.

Reply:
fixed